src/api/script/delayed_job_stats.rb

```diff
@@ -1,6 +1,22 @@
 puts "-----------EVENT-STATS------------"
-event_classes = [Event::Package, Event::Project]
-event_types = event_classes.flat_map(&:descendants).map(&:name)
+event_types = ["Event::BranchCommand",
+               "Event::Build",
+               "Event::CommentForPackage",
+               "Event::Commit",
+               "Event::CreatePackage",
+               "Event::DeletePackage",
+               "Event::ServiceFail",
+               "Event::ServiceSuccess",
+               "Event::UndeletePackage",
+               "Event::UpdatePackage",
+               "Event::Upload",
+               "Event::VersionChange",
+               "Event::CommentForProject",
+               "Event::CreateProject",
+               "Event::DeleteProject",
+               "Event::UndeleteProject",
+               "Event::UpdateProjectConfig",
+               "Event::UpdateProject"]
```

Comment:
Wouldn't the best thing here be to list all subclasses of `Event`? Otherwise we have to update this file if we add a new Event class.

Reply:
But it is not all of the subclasses of Event. This is only the subclasses of the old `Event::Package` and `Event::Project`. So now, the classes in `Event` related to `Project` and `Package`. I think the only way is to listing they directly.

Reply:
@Ana06 Please add a comment, mentioning those are only the subclasses  related to Project and Package

Reply:
@bgeuken ok! 👍